### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by sumairq**
## Added the following functionality

- Create tasks
- Delete tasks
- Edit Tasks

### CODE REVIEW

File: src/modules/todoCollection.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  }
+
+  create(data) {
+    this.list.push(data);
+    this.display(data);
+    this.arrange();
+    this.remove();
+    this.edit();
+    this.markComplete();
+    this.setStorage();
```
Comment (hiromataba) [inline src/modules/todoCollection.js:15]: - [ ] Please use normal functions for this project as mentioned in the project requirement, not classes, Kindly replace your classes with functions and follow the project requirement to implement the required functionalities 

image

File: src/modules/item.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export default class Item {
+  constructor(index, description, completed) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index; // an intuition
+  }
+}
```
Comment (hiromataba) [inline src/modules/item.js]: - [ ] Please use normal functions for this project as mentioned in the project requirement, not classes, Kindly replace your classes with functions and follow the project requirement to implement the required functionalities 

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi :wave:  @sumairq , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :champagne: 
- Descriptive README file :heavy_check_mark: 
- Your linters are passing :+1: 

## Required Changes ♻️

- [ ] Since you are working with webpack, I think it would be better to add the commands in your README file to install npm `npm install` and to start the server `npm start``

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @sumairq ,  

Your project is complete! Congratulations!  🎉💯🌟 
Everything is up and running!



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**